### PR TITLE
 FLEX-3830 ~ Implements communication with ABB RTU-540.

### PR DESCRIPTION
**Pull request by kevinsmeets**

### CODE REVIEW

File: osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/infra/networking/helper/DataAttribute.java [hunk 1]
```diff
@@ -55,6 +55,10 @@
      */
     RCB_A("rcb_A"),
     /**
+     * Property of LLN0 Node, rcb_A, contains the buffered report information.
```
Comment (rlemmers) [inline osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/infra/networking/helper/DataAttribute.java]: rcb_A or rcb_A01? Should we mention this is specific for the LMD RTU?

File: osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/infra/networking/services/commands/Iec61850EnableReportingCommand.java [hunk 2]
```diff
@@ -162,9 +169,11 @@ public void enableUnbufferedReportingOnLightMeasurementDevice(final Iec61850Clie
         final String dataSetReference = reporting.getString(SubDataAttribute.DATA_SET);
         LOGGER.info("dataSetReference for unbuffered reporting: {}", dataSetReference);
 
-        // Set data set reference.
-        reporting.writeString(SubDataAttribute.DATA_SET, "TEMPLATELD0/LLN0.StatNrmlA");
-        reporting.writeString(SubDataAttribute.REPORT_ID, "A");
+        if (StringUtils.isEmpty(dataSetReference)) {
+            // Set data set reference.
+            reporting.writeString(SubDataAttribute.DATA_SET, "AA1TH01LD0/LLN0.StatNrmlA");
```
Comment (rlemmers) [inline osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/infra/networking/services/commands/Iec61850EnableReportingCommand.java]: Can't we reuse the dynamic code from line 123?
Line 123/124: final String dataSet = IED.ABB_RTU.getDescription() + LogicalDevice.LD0.getDescription() + "/" + LogicalNode.LOGICAL_NODE_ZERO.getDescription() + "." + "StatNrmlA";

Comment (kevinsmeets) [reply osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/infra/networking/services/commands/Iec61850EnableReportingCommand.java]: Refactored into private function and used the new function in both enableBufferedReportingOnLightMeasurementDevice() and enableUnbufferedReportingOnLightMeasurementDevice().

File: osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/infra/networking/helper/DataAttribute.java (hunk 1 above)
Comment (kevinsmeets) [reply osgp-protocol-adapter-iec61850/src/main/java/com/alliander/osgp/adapter/protocol/iec61850/infra/networking/helper/DataAttribute.java]: Removed rcb_A01. Both data attributes are the same, but we are using only data attribute rcb_A.

**Pull request merged**